### ISSUE TITLE
Plot how the algorithm converges

### ISSUE THREAD

**Issue by uyot10** [tpaviot/ProcessScheduler#119]
Is there any way to plot how the algorithm converges? It would be nice to see how much it improves in each iteration

**Comment by uyot10** [comment]
When the algorithm finishes and finds the optimal value it prints how many iterations it has done. It would be useful to plot how the algorithm improved the value in each iteration to see which iterations are the ones that matter the most and in a more visual way.

**Comment by tpaviot** [comment]
right, in the optimization mode that's an info that can be useful. In a graphical way similar to the loss in an ML training process, right?

**Comment by uyot10** [comment]
Exactly, so it would be great to implement a simple graph to show the improvement on each iteration.
